XRTK-Core/Packages/com.xrtk.core/Inspectors/Extensions/ScriptableObjectExtensions.cs

```diff
@@ -63,7 +65,7 @@ public static ScriptableObject CreateAsset(this ScriptableObject scriptableObjec
                 Directory.CreateDirectory(Path.GetFullPath(path));
             }
 
-            path = path.Replace(".asset", string.Empty);
+            path = path.Replace($"{Directory.GetParent(Application.dataPath).FullName}\\", string.Empty);
```

Comment:
These changes need to get pulled out too. I thought I had already but I might have botch that one up